experiments/AtmosLES/taylor-green.jl

```diff
@@ -125,7 +119,7 @@ function config_greenvortex(
         solver_method = LSRK144NiegemannDiehlBusch,
     )
 
-    _C_smag = FT(C_smag(param_set))
+    _C_smag = FT(FT(0.21))
```

Comment:
Why this change? We should absolutely use `CLIMAParameters` for this; if there's a problem, let's fix it.

Reply:
I could not seem to get this work out of the box, would be glad if you knew how to resolve this.

Reply:
Let's discuss what needs to be done on with regards to the CI on Slack

Reply:
See suggestion above.